promql/printer.go

```diff
@@ -174,7 +175,11 @@ func (node *MatrixSelector) String() string {
 		Name:          node.Name,
 		LabelMatchers: node.LabelMatchers,
 	}
-	return fmt.Sprintf("%s[%s]", vecSelector.String(), strutil.DurationToString(node.Range))
+	offset := ""
+	if node.Offset != time.Duration(0) {
+		offset = fmt.Sprintf(" offset %s", strutil.DurationToString(node.Offset))
```

Comment:
We capitalize all the other printed keywords. We should change them all to lower-case, but maybe better in a separate change. So here, maybe capitalize it for consistency?